app/javascript/schools/courses/components/course_certificates/CourseCertificates__EditDrawer.re

```diff
@@ -245,6 +251,16 @@ let make =
                 {str("No")}
               </button>
             </div>
+            {!canBeAutoIssued
+               ? <div
+                   className="border border-yellow-400 rounded bg-yellow-400 py-2 px-3 my-1">
+                   <i className="fas fa-exclamation-triangle" />
+                   <span className="ml-2">
+                     "Create a milestone target in the last level to auto-issue the certificate"
```

Comment:
It's good to have a more explicit message in here something along the lines:

_Please note that the last level of this course does not have any milestone targets. This certificate will be auto-issued only if the last level has at least one milestone target._